app/src/main/res/drawable/ic_more_vert_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```

Comment:
I think changing the three dot menu color to white might look better

Reply:
To change color to white, I think we should change the source of our vector image.
Because we added as a background image, change color  doesn't work.

Reply:
I will show you how to invert colors 😄